research_constantin.ipynb

```diff
@@ -7,6 +7,13 @@
     "# This is Constantin research notebook for the project smart city"
```

Comment:
<p>Hey i thing you need to delete that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ticuss/Smart-City-Project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>